### ISSUE TITLE
Error in createPaymentMethod: Card details not complete (Android)

### ISSUE THREAD

**Issue by fullstack17** [stripe/stripe-react-native#1192]
**Describe the bug**

**To Reproduce**
1. on CardField fields, I put 5200 8282 8282 8210 12/34 123
2. Click on Confirm button
3. It calls Stripe Method - createPaymentMethod
4.  on iOS version, it works fine, but on Android it returns error: 
 ```
  error: {
    code: "Failed",
    declineCode: null,
    localizedMessage: "Card details not complete",
    message: "Card details not complete",
    stripeErrorCode: null,
    type: null
  }
 ```

**Expected behavior**
Android version should work as same as iOS version

**Smartphone (please complete the following information):**
 - Device: Android emulator Pixel API 31
 - Version Android 12

**Environment**
"@stripe/stripe-react-native": "0.14.0",
"react-native": "0.64.3",

**Comment by ludwig-pro** [comment]
Hey! 

I have precisely the same issue. I reproduce it only during the E2E test with Detox on CI. I can't reproduce it manually with the same input.

```js
// method who throw
        const { error } = await stripeConfirmSetupIntent(clientSecret, {
          paymentMethodType: 'Card',
        });
```

**Environment**
- "@stripe/stripe-react-native": "0.17.x" / "0.18.x" / "0.19.x"
- "react-native": "0.67.5"

**Smartphone:**
- Galaxy Nexus API 30

**Image**
![image](https://user-images.githubusercontent.com/62896243/204355773-fe718987-f532-4a14-8d8d-4120b609d18e.png)

**Video**
https://user-images.githubusercontent.com/62896243/204356092-b3891d98-90f4-43aa-8719-3339fa992afc.mp4

**Comment by ludwig-pro** [comment]
❗<= this symbol appears before we try to submit the card data ?!

**Comment by fullstack17** [comment]
@ludwig-pro Yes, once the card field is filled, then `❗` symbol is appeared.

**Comment by ludwig-pro** [comment]
Do you know the meaning? did you find a solution? (I think we'll have to patch in Kotlin something) 

We will contact the support of stripe directly next week about this one.

**Comment by fullstack17** [comment]
@ludwig-pro Yes, right. it happens only on android, I'm not sure why this is happening on only android, not iOS.

**Comment by charliecruzan-stripe** [comment]
This was fixed in v0.22.0, let me know if you're still running into it!
